shared/chat/conversations-list/index.desktop.js

```diff
@@ -64,7 +69,7 @@ const TopLine = ({hasUnread, showBold, participants, subColor, timestamp, userna
             inline={true}
             type='BodySemibold'
             style={{...boldOverride, color: usernameColor}}
-            commaColor={commaColor}
+            plainText={true}
```

Comment:
this creates a single text instead of a bunch of nested texts which are hard to optimize